molecule/base_molecule.h

```diff
@@ -306,6 +312,21 @@ namespace indigo
         bool isBondHighlighted(int idx);
         void highlightSubmolecule(BaseMolecule& sub, const int* mapping, bool entire);
 
+		void unselectAll();
+		void selectAtom(int idx);
+		void selectBond(int idx);
+		void selectAtoms(const Filter& filter);
+		void selectBonds(const Filter& filter);
+		void unselectAtom(int idx);
+		void unselectBond(int idx);
+		int countSelectedAtoms();
+		int countSelectedBonds();
+		bool hasSelection();
+		bool isAtomSelected(int idx);
+		bool isBondSelected(int idx);
+		void selectSubmolecule(BaseMolecule& sub, const int* mapping, bool entire);
```

Comment:
Why there are tabs instead spaces?
This happens with this commit: fe9aefb91d8e5e7bb7448bda50fe196f61774591 on 12.01.2020.
Probably, clang autoformatting was disabled?

Reply:
fixed